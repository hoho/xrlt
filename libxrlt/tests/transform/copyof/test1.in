id:0, type:100, last:0, error: 0, data:
id:0, type:100, last:0, error: 0, data:
id:1, type:600, last:1, error: 0, data:<r><a>b</a><c>d</c><e>f</e></r>
id:0, type:100, last:0, error: 0, data:
