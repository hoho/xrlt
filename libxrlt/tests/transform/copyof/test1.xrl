<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">

    <xrl:response>
        <xrl:variable name="tmp">
            <xrl:include>
                <xrl:href>/tmp</xrl:href>
                <xrl:type>xml</xrl:type>
            </xrl:include>
        </xrl:variable>

        <xrl:variable name="tmp2">
            <a>aaa</a>
            <b>bbb</b>
            <c>ccc</c>
            <d>ddd</d>
        </xrl:variable>

        <xrl:copy-of select="$tmp/r/*[local-name() = 'a' or local-name() = 'e']" />
        <xrl:copy-of select="$tmp2/*[local-name() = 'a' or local-name() = 'd']" />
    </xrl:response>

</xrl:requestsheet>
