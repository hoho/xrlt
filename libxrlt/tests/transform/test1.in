id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:3, type:2, last:0, error: 0, data:{"aaa
id:3, type:2, last:1, error: 0, data:bbb": 123, "ccc": true}
id:0, type:0, last:0, error: 0, data:
id:2, type:2, last:0, error: 0, data:<hello>
id:2, type:2, last:0, error: 0, data:world
id:2, type:2, last:1, error: 0, data:</hello>
id:0, type:0, last:0, error: 0, data:
id:1, type:2, last:0, error: 0, data:there are
id:1, type:2, last:0, error: 0, data: many clouds
id:1, type:2, last:1, error: 0, data: in the sky
id:0, type:0, last:0, error: 0, data:
