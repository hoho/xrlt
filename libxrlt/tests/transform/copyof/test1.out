XRLT_STATUS_SUBREQUEST
sr id: 1
sr method: GET
sr type: XML
sr url: /tmp
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: b
chunk: f
chunk: aaa
chunk: ddd
XRLT_STATUS_DONE
